ci/cirrus_wheels.yml

```diff
@@ -28,14 +28,7 @@ cirrus_wheels_linux_aarch64_task:
     - env:
         CIBW_BUILD: cp311-manylinux* cp312-manylinux*
   env:
-    CIBW_PRERELEASE_PYTHONS: True
-    # TODO remove the CIBW_BEFORE_BUILD_LINUX line once there are numpy2.0 wheels available on PyPI.
-    # Also remove/comment out PIP_NO_BUILD_ISOLATION, PIP_EXTRA_INDEX_URL flags.
-    CIBW_BEFORE_BUILD_LINUX: "pip install numpy>=2.0.0.dev0 meson-python cython pythran pybind11 ninja;bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
-    CIBW_ENVIRONMENT: >
-      PIP_PRE=1
-      PIP_EXTRA_INDEX_URL=https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
-      PIP_NO_BUILD_ISOLATION=false
+    CIBW_PRERELEASE_PYTHONS: False
```

Comment:
could `main` be adjusted so I don't need to repeat this or no?

Reply:
Yes, I think so. This same thing was already removed from `.github/workflows/wheels.yml` in gh-20585.